kt/godot-coroutine-library/build.gradle.kts

```diff
@@ -0,0 +1,63 @@
+import versioninfo.fullGodotKotlinJvmVersion
+
+plugins {
+    alias(libs.plugins.kotlin.jvm)
+    id("com.utopia-rise.api-generator")
+    id("com.utopia-rise.godot-publish")
+    id("com.utopia-rise.versioninfo")
+    alias(libs.plugins.kotlinPreProcessors)
+}
+
+val isRelease = project.hasProperty("release")
+
+kotlinDefinitions {
+    definitionsObjectName.set("GodotJvmBuildConfig")
+
+    define("DEBUG", !isRelease)
+}
+
+
+kotlin {
+    jvmToolchain(11)
+}
+
+java {
+    withSourcesJar()
+    withJavadocJar()
+}
+
+dependencies {
+    compileOnly(project(":godot-library"))
+    implementation("com.utopia-rise:tools-common:$fullGodotKotlinJvmVersion")
+    api("org.jetbrains.kotlinx:kotlinx-coroutines-core:${libs.versions.kotlinCoroutine.get()}")
+}
+
+tasks {
+    compileKotlin {
+        dependsOn(":godot-library:generateAPI")
+    }
+
+    // here so the sourcesJar task has an explicit dependency on the generateApi task. Needed since gradle 8
+    withType<Jar> {
+        dependsOn(":godot-library:generateAPI")
+    }
+}
```

Comment:
I tried to remove it but it doesn't want to build without that.

I get the following error:

What went wrong:
A problem was found with the configuration of task ':kt:godot-coroutine-library:sourcesJar' (type 'Jar').
- Gradle detected a problem with the following location: 'D:\Godot\Module\kotlin\modules\kotlin_jvm\kt\godot-coroutine-library\src\main\kotlin'.

  Reason: Task ':kt:godot-coroutine-library:sourcesJar' uses this output of task ':kt:godot-library:generateAPI' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.

  Possible solutions:
    1. Declare task ':kt:godot-library:generateAPI' as an input of ':kt:godot-coroutine-library:sourcesJar'.
    2. Declare an explicit dependency on ':kt:godot-library:generateAPI' from ':kt:godot-coroutine-library:sourcesJar' using Task#dependsOn.
    3. Declare an explicit dependency on ':kt:godot-library:generateAPI' from ':kt:godot-coroutine-library:sourcesJar' using Task#mustRunAfter.